Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -2021,8 +2021,7 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
             setConstant(node, jsBoolean(std::isnan(value.asNumber())));
             break;
         }
-        if (node->child1().useKind() != DoubleRepUse)
-            clobberWorld();
+        clobberWorld();
```

Comment:
Add an assertion for `node->child1().useKind() == UntypedUse`?

Reply:
Added.